src/pages/resultsView/expression/expressionHelpers.tsx

```diff
@@ -204,4 +213,24 @@ export function expressionTooltip(d:IBoxScatterPlotPoint, studyIdToStudy:{[study
             )}
         </div>
     );
+}
+
+export function getPossibleRNASeqVersions(expressionProfiles:Pick<MolecularProfile, "molecularProfileId">[]) {
+    const possibleRNASeqVersions: { [id: string]: boolean } = {
+        'rna_seq_mrna': false,
+        'rna_seq_v2_mrna': false
+    };
+
+    for (const expressionProfile of expressionProfiles) {
```

Comment:
i would just use two calls to _.some, one for each rnaseq version.   there are not a lot of expressionProfiles so clarity is more important that performance.